src/main/java/oshi/hardware/platform/linux/LinuxGlobalMemory.java

```diff
@@ -121,6 +122,51 @@ public long getTotal() {
         }
         return this.totalMemory;
     }
+    
+    @Override
+    public long getSwapAvailable() {
+        long swapAvailable = 0;
+        List<String> memInfo = null;
+        
+        try {
+            memInfo = FileUtil.readFile("/proc/meminfo");

```

Comment:
You can actually get swap free and total from the sysinfo structure (that's where /proc/meminfo gets it).  See `getTotal()` for a template; you can make the swap total a peristent class variable to save repeated reads.